lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -120,4 +120,14 @@ void rtc_clear_wakeup_flag(void)
 	RTC_ISR &= ~RTC_ISR_WUTF;
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Clears the Alarm A flag
+
+@details RTC_ISR[13:8] bits are not write protected: no need to unlock
+*/
+void rtc_clear_alarm_flag(void)
+{
+	RTC_ISR &= ~RTC_ISR_ALRAF;
```

Comment:
You've added this to the common code.  Either, 

1. The common code should clear bits 8 and 9 (Alarm A and B)  (On f0, it's "must be kept at reset value (0)" so that's "safe") (and docs updated to indicate a bit more clearly what it's doing)
2. Or, the function prototype stays in common code, and the implementation is moved to just f0, where it only clears Alarm A, and implementations added to L1, F2, F4 (and later eventually probably  l0, f3, f7.....)

I'm inclined to go with option 1,  but as it stands, this would be misleading for the  non-f0 parts.  Opinions? Thoughts?

 I guess there's also 

3. Separate functions for clear alarm A and clear alarm B
4. Use a function more like the F1 API, which uses a single "clear flag" function and takes the flag to clear as an arg.

(I would still lean towards option 1 personally)